### DISCUSSION TITLE
About theme color variable couldn't work in Shadow syntax.

### DISCUSSION THREAD

**Discussion by JohnYang02182** [unocss/unocss#2407] [Q&A] [answered]
I attempted to use a theme color variable in the shadow syntax, but it didn't seem to work. I would greatly appreciate it if someone could inform me of what I might have overlooked.

[playground](https://uno.antfu.me/play/?html=DwEwlgbgBAFgtAMwK4BsVQC4FMAeG4DGWAdtgE5QIq5QDOW1B%2BxA9sVlALYCGcAjAAYAfACgoUUJCgEU3WrQC8AIgJwA7jDDYoOFAC5seOLq4gDufABYTBJLQwtOcMqg60Y3ECzVwA2gIAHHAB9OAAmIODLSIFggLIwHjIATwBdJSEAFSx7Dk4scG4oAEckLASc4AB6cAhRatqhIA&config=JYWwDg9gTgLgBAbzgEwKYDNgDtUGEJaYDmcAvnOlBCHAOQCuWEAxgM6u0BQnqAHpLBQYAhvQA28NJhz5CwIgAoEnOHBgALVCFQAuRCtVxmEMdFZ7lhw7TBRQwqAE9ae2lFTIuV0gdIAaAyhxVHM4AG0DAF1OUgBKIA&css=Q&options=N4IgzgLgTglgxhEAuaBXApgXyA)

**Comment by zyyv** [comment, marked as answer]
In the shadow rules, if you use the bracket syntax, we will treat the shadow as your custom control fully, and will not further analyze the color or other.
